Installation issue: aml versions master & develop

### Steps to reproduce the issue

```console
$ spack install aml@master
==> aml: Executing phase: 'build'
==> Error: ProcessError: Command exited with status 2:
    'make' '-j16'

33 errors found in build log:
     240    libtool: compile:  /g/g0/butler59/spack/lib/spack/env/gcc/gcc -std=gnu99 -DHAVE_CONFIG_H -I. -I
            /var/tmp/butler59/spack-stage/spack-stage-aml-master-uwns4unv4wq4g5kveq7l5t4pdzkzazly/spack-src
            /src -I../include -I/var/tmp/butler59/spack-stage/spack-stage-aml-master-uwns4unv4wq4g5kveq7l5t
            4pdzkzazly/spack-src/include -I/var/tmp/butler59/spack-stage/spack-stage-aml-master-uwns4unv4wq
            4g5kveq7l5t4pdzkzazly/spack-src/excit/src -pthread -fopenmp -g -O2 -MT tiling/tiling_resize.lo 
            -MD -MP -MF tiling/.deps/tiling_resize.Tpo -c tiling/tiling_resize.c  -fPIC -DPIC -o tiling/.li
            bs/tiling_resize.o
     241    libtool: compile:  /g/g0/butler59/spack/lib/spack/env/gcc/gcc -std=gnu99 -DHAVE_CONFIG_H -I. -I
            /var/tmp/butler59/spack-stage/spack-stage-aml-master-uwns4unv4wq4g5kveq7l5t4pdzkzazly/spack-src
            /src -I../include -I/var/tmp/butler59/spack-stage/spack-stage-aml-master-uwns4unv4wq4g5kveq7l5t
            4pdzkzazly/spack-src/include -I/var/tmp/butler59/spack-stage/spack-stage-aml-master-uwns4unv4wq
            4g5kveq7l5t4pdzkzazly/spack-src/excit/src -pthread -fopenmp -g -O2 -MT utils/bitmap.lo -MD -MP 
            -MF utils/.deps/bitmap.Tpo -c utils/bitmap.c  -fPIC -DPIC -o utils/.libs/bitmap.o
     242    libtool: compile:  /g/g0/butler59/spack/lib/spack/env/gcc/gcc -std=gnu99 -DHAVE_CONFIG_H -I. -I
            /var/tmp/butler59/spack-stage/spack-stage-aml-master-uwns4unv4wq4g5kveq7l5t4pdzkzazly/spack-src
            /src -I../include -I/var/tmp/butler59/spack-stage/spack-stage-aml-master-uwns4unv4wq4g5kveq7l5t
            4pdzkzazly/spack-src/include -I/var/tmp/butler59/spack-stage/spack-stage-aml-master-uwns4unv4wq
            4g5kveq7l5t4pdzkzazly/spack-src/excit/src -pthread -fopenmp -g -O2 -MT utils/error.lo -MD -MP -
            MF utils/.deps/error.Tpo -c utils/error.c  -fPIC -DPIC -o utils/.libs/error.o
     243    libtool: compile:  /g/g0/butler59/spack/lib/spack/env/gcc/gcc -std=gnu99 -DHAVE_CONFIG_H -I. -I
            /var/tmp/butler59/spack-stage/spack-stage-aml-master-uwns4unv4wq4g5kveq7l5t4pdzkzazly/spack-src
            /src -I../include -I/var/tmp/butler59/spack-stage/spack-stage-aml-master-uwns4unv4wq4g5kveq7l5t
            4pdzkzazly/spack-src/include -I/var/tmp/butler59/spack-stage/spack-stage-aml-master-uwns4unv4wq
            4g5kveq7l5t4pdzkzazly/spack-src/excit/src -pthread -fopenmp -g -O2 -MT layout/reshape.lo -MD -M
            P -MF layout/.deps/reshape.Tpo -c layout/reshape.c  -fPIC -DPIC -o layout/.libs/reshape.o
     244    libtool: compile:  /g/g0/butler59/spack/lib/spack/env/gcc/gcc -std=gnu99 -DHAVE_CONFIG_H -I. -I
            /var/tmp/butler59/spack-stage/spack-stage-aml-master-uwns4unv4wq4g5kveq7l5t4pdzkzazly/spack-src
            /src -I../include -I/var/tmp/butler59/spack-stage/spack-stage-aml-master-uwns4unv4wq4g5kveq7l5t
            4pdzkzazly/spack-src/include -I/var/tmp/butler59/spack-stage/spack-stage-aml-master-uwns4unv4wq
            4g5kveq7l5t4pdzkzazly/spack-src/excit/src -pthread -fopenmp -g -O2 -MT utils/vector.lo -MD -MP 
            -MF utils/.deps/vector.Tpo -c utils/vector.c  -fPIC -DPIC -o utils/.libs/vector.o
     245    In file included from area/linux.c:12:0:
  >> 246    ../include/aml.h:23:19: fatal error: excit.h: No such file or directory
     247     #include <excit.h>
     248                       ^
     249    compilation terminated.
  >> 250    make[1]: *** [area/linux.lo] Error 1
     251    make[1]: *** Waiting for unfinished jobs....
     252    In file included from dma/dma.c:11:0:
     ...
```
This is the same output for 

```console
$ spack install aml@develop
```


### Information on your system

<!-- Please include the output of `spack debug report` -->

```
* **Spack:** 0.15.4-624-ebe5a5652c
* **Python:** 3.7.2
* **Platform:** linux-rhel7-broadwell
```


### General information

<!-- These boxes can be checked by replacing [ ] with [x] or by clicking them after submitting the issue. -->
- [ ] I have run `spack debug report` and reported the version of Spack/Python/Platform
- [ ] I have run `spack maintainers <name-of-the-package>` and @mentioned any maintainers
- [ ] I have uploaded the build log and environment files
- [ ] I have searched the issues of this repo and believe this is not a duplicate

--- a/var/spack/repos/builtin/packages/aml/package.py
+++ b/var/spack/repos/builtin/packages/aml/package.py
@@ -12,9 +12,10 @@ class Aml(AutotoolsPackage):
     homepage = "https://xgitlab.cels.anl.gov/argo/aml"
     url = "https://www.mcs.anl.gov/research/projects/argo/downloads/aml-0.1.0.tar.gz"
     git = "https://xgitlab.cels.anl.gov/argo/aml.git"
+
     version('0.1.0', sha256='cc89a8768693f1f11539378b21cdca9f0ce3fc5cb564f9b3e4154a051dcea69b')
-    version('develop', branch='staging')
-    version('master', branch='master')
+    version('develop', branch='staging', submodules=True)
+    version('master', branch='master', submodules=True)
 
     depends_on('numactl')
 

